srcpkgs/hunspell-el_GR/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'hunspell-el_GR'
+pkgname=hunspell-el_GR
+version=0.4.0
+revision=1
+_version="${version}-87"
+archs=noarch
+build_style=fetch
+hostmakedepends="unzip"
+short_desc="Greek (Greece) dictionary for hunspell"
+maintainer="Protesilaos Stavrou <info@protesilaos.com>"
+license="GPL-3"
+homepage="https://sourceforge.net/projects/orthos-spell"
+distfiles="${homepage}/files/latest/download"
+checksum=541328be3043d861361fd8d79874ac8d4cd042b657bbd9ba94a5dafab9cbae2d
+
+post_extract() {
```

Comment:
extracting was organized better in that PR.

Reply:
still

Reply:
Okay, I will follow that method.